service-base/src/main/java/org/eclipse/hono/service/HealthCheckServer.java

```diff
@@ -87,6 +107,60 @@ private void prepareHealthCheck() {
         }
     }
 
+    private void registerConsumers() {
+        readinessConsumer = vertx.eventBus().consumer(EVENT_BUS_ADDRESS_HEALTH_READINESS);
+        readinessConsumer.handler(this::processReadinessMessage);
+        LOG.info("listening on event bus [address: {}] for readiness status", EVENT_BUS_ADDRESS_HEALTH_READINESS);
+
+        livenessConsumer = vertx.eventBus().consumer(EVENT_BUS_ADDRESS_HEALTH_LIVENESS);
+        livenessConsumer.handler(this::processLivenessMessage);
+        LOG.info("listening on event bus [address: {}] for liveness status", EVENT_BUS_ADDRESS_HEALTH_LIVENESS);
+    }
+
+    private void processReadinessMessage(final Message<JsonObject> msg) {
+
+        final String verticleName = msg.headers().get(MSG_FIELD_VERTICLE_NAME);
+        final String instanceId = msg.headers().get(MSG_FIELD_VERTICLE_INSTANCE_ID);
+        final String procedureName = msg.headers().get(MSG_FIELD_PROCEDURE_NAME);
+        final Status status = new Status(msg.body());
+
+        LOG.info("received readiness update {}/{}/{}: {}", verticleName, instanceId, procedureName, status.isOk());
```

Comment:
IMHO DEBUG should be enough. in the end, the readiness will be monitored by an external system that uses this information. So, if anybody is interested in getting notified about a change, they will probably do so based on the external monitoring system, right?